src/main/jbake/content/blog/2016/09/06/consuming-hawkular-api-over-ssl.adoc

```diff
@@ -0,0 +1,84 @@
+= Consuming Hawkular API over SSL with self signed certificates
+Josejulio Martínez
+2016-9-6
+:jbake-type: post
+:jbake-status: published
+:jbake-tags: blog
+
+In this post I'll show you how to trust self-signed certificates for use with Ruby client, used on projects such as ManageIQ and HawkFX.
```

Comment:
It would be nice to mention on what operating system you tested this or is supposed to work.